test/integration/features/basic.feature

```diff
@@ -10,6 +10,42 @@ Feature: Basic
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
 
+  Scenario: OpenShift is ready after startup
+    After startup of Minishift OpenShift instance should respond correctly on its html endpoints
+    and OpenShift web console should be accessible.
+    Given Minishift has state "Running"
+     Then body of HTTP request to "OpenShift" at "/healthz" contains "ok"
```

Comment:
Technically though, I think we should be looking at the returned status code:

    Then response code of HTTP request to "OpenShift" at "/healthz" equals "200"

See also https://docs.openshift.org/latest/dev_guide/application_health.html. AFAIU there is no need to return anything in the body. It is the return code which matters. I recommend to add an appropriate  step and update the test.